bolt/test/runtime/X86/exceptions-lpstart-zero.s

```diff
@@ -0,0 +1,107 @@
+# RUN: %clangxx %cflags -no-pie %s -o %t.exe -Wl,-q
+# RUN: llvm-bolt %t.exe -o %t.exe.bolt
+# RUN: %t.exe.bolt
+
+# REQUIRES: system-linux
+
+## Test that BOLT properly handles LPStart when LPStartEncoding is different
+## from DW_EH_PE_omit.
+
+# Test case compiled from:
+#
+# int main() {
```

Comment:
what was the build command to produce main code and exception table?

Reply:
`clang input -o output.s -S`